<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeasuringChannelSelect</class>
 <widget class="QDialog" name="MeasuringChannelSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>911</width>
     <height>561</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1">
        <property name="margin">
         <number>6</number>
        </property>
        <item>
         <widget class="QTreeWidget" name="twObjects">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="expandsOnDoubleClick">
           <bool>false</bool>
          </property>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="cmdSelect">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/icons/images/forward.png</normaloff>:/icons/images/forward.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cmdDeselect">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/icons/images/deselect.png</normaloff>:/icons/images/deselect.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="twSelected">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::MultiSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>20</number>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="cmdSave">
          <property name="text">
           <string>Применить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Отмена</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cmdSelect</sender>
   <signal>clicked()</signal>
   <receiver>MeasuringChannelSelect</receiver>
   <slot>itemSelect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>85</x>
     <y>628</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdDeselect</sender>
   <signal>clicked()</signal>
   <receiver>MeasuringChannelSelect</receiver>
   <slot>itemRemove()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>628</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MeasuringChannelSelect</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>876</x>
     <y>573</y>
    </hint>
    <hint type="destinationlabel">
     <x>483</x>
     <y>830</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdSave</sender>
   <signal>clicked()</signal>
   <receiver>MeasuringChannelSelect</receiver>
   <slot>saveChanges()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>786</x>
     <y>566</y>
    </hint>
    <hint type="destinationlabel">
     <x>502</x>
     <y>785</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>itemSelect()</slot>
  <slot>itemRemove()</slot>
  <slot>saveChanges()</slot>
 </slots>
</ui>
